frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -244,6 +245,21 @@ const getAnnotation = (obj: K8sResourceKind, annotation: string): string => {
   return _.get(obj, ['metadata', 'annotations', annotation]);
 };
 
+export const mapAnnotationsToJSON = (
+  annotations: ObjectMetadata['annotations'],
+  annotationKeys: string[],
```

Comment:
I think a better API is just to take in a single annotation rather than a list, which is the common case. I wouldn't expect us to need to parse more than a couple annotations, so it's not difficult to call the helper more than once.

Reply:
Fixed